
class CrawlerError(Exception):
    print("Folgende Fehler sind aufgetreten:")
    pass
